### ISSUE TITLE
Can you explain these ideas ?

### ISSUE THREAD

**Issue by moeenio** [MasterOfTheTiger/reimagine-scratch#5]
Can you explain these ideas ?
I'd gladly work on the button style :)
![Screenshot from this repo's projects tab, with two cards : "New button style" and "Zoom on click of profile and project icons"](https://user-images.githubusercontent.com/37651007/64876911-8cd4e100-d650-11e9-9a49-7cfe54dff510.png)

**Comment by MasterOfTheTiger** [comment]
It was an idea to redesign every "old" button on Scratch to something more flashy'

**Comment by moeenio** [comment]
And the second one ?

**Comment by MasterOfTheTiger** [comment]
That one was for clicking other people's profile icons, and it would pop up with more information about the person as I recall. I don't remember very much about it. It is a complex idea.

**Comment by MasterOfTheTiger** [comment]
The biggest priority should probably be to get this working on the new pages as well, that way we can have a consistent design.

**Comment by moeenio** [comment]
Get what working on the new pages ?

**Comment by MasterOfTheTiger** [comment]
Sorry. I mean get this custom header working on all pages.

**Comment by moeenio** [comment]
Ok. May I work on the button style ?

**Comment by MasterOfTheTiger** [comment]
Yes, you may.

**Comment by moeenio** [comment]
What style do you want ? like on scratch new pages ?

**Comment by MasterOfTheTiger** [comment]
Yes. Anything that makes it look fluid with the rest. Also keep in mind more features which may be interesting but not difficult to add.

**Comment by moeenio** [comment]
Why did I just got a notification ? Nothing happened

**Comment by MasterOfTheTiger** [comment]
Weird.
